docs/guide/styles.md

```diff
@@ -1,15 +1,320 @@
+
+Here's how you would set the screen background to lime:
+
+```python
+widget.styles.background = "lime"
+```
+
+In addition to color names, you can also use any of the following ways of expressing a color:
+
+- RGB hex colors starts with a `#` followed by three pairs of one or two hex digits; one for the red, green, and blue color components. For example, `#f00` is an intense red color, and `#9932CC` is *dark orchid*.
+- RGB decimal color start with `rgb` followed by a tuple of three numbers in the range 0 to 255. For example `rgb(255,0,0)` is intense red, and `rgb(153,50,204)` is *dark orchid*.
+- HSL colors start with `hsl` followed by a angle between 0 and 360 and two percentage values, representing Hue, Saturation and Lightness. For example `hsl(0,100%,50%)` is intense red and `hsl(280,60%,49%)` is *dark orchid*
+
+
+The background and color styles will also accept a [color][textual.color.Color] object which can be used to create colors dynamically.
+
+The following example adds three widgets and sets color styles.
+
+```python title="colors01.py" hl_lines="16-19"
+--8<-- "docs/examples/guide/styles/colors01.py"
+```
+
+Here is the output:
+
+```{.textual path="docs/examples/guide/styles/colors01.py"}
+```
+
+### Alpha
+
+Textual represents color internally as a tuple of three values for the red, green, and blue components.
+
+Textual support a common fourth value called *alpha* which can make a color translucent. If you set alpha on a background color, Textual will blend the background with the color beneath it. If you set alpha on the text color, then Textual will blend the text with the background color.
+
+There are a few ways you can set alpha on a color in Textual.
+
+- You can set the alpha value of a color by adding a fourth digit or pair of digits to a hex color. The extra digits form an alpha component which ranges from 0 for completely transparent to 255 (completely opaque). Any value between 0 and 255 will be translucent. For example `"#9932CC7f"` is a dark orchid which is roughly 50% translucent.
+- You can also set alpha with the `rgba` format, which is identical to `rgb` with the additional of a fourth value that should be between 0 and 1, where 0 is invisible and 1 is opaque. For example `"rgba(192,78,96,0.5)"`.
+- You can add the `a` parameter on a [Color][textual.color.Color] object. For example `Color(192, 78, 96, a=0.5)` creates a translucent dark orchid. 
+
+The following examples shows what happens when you set alpha on background colors:
+
+```python title="colors01.py" hl_lines="12-15"
+--8<-- "docs/examples/guide/styles/colors02.py"
+```
+
+Notice that an alpha of 0.1 the background almost matches the screen, but at 1.0 it is a solid color. 
+
+```{.textual path="docs/examples/guide/styles/colors02.py"}
+```
+
+## Dimensions
+
+Widgets occupy a rectangular region of the screen, which may be as small as a single character or as large as the screen. Potentially *larger* if [scrolling](#) is enabled.
```

Comment:
Take it or leave it: "Potentially larger if scrolling is enabled." doesn't sound right to me as a standalone sentence.

```suggestion
Widgets occupy a rectangular region of the screen, which may be as small as a single character or as large as the screen. If [scrolling](#) is enabled, widgets can be even larger than the screen.
```